.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+### Platforms Impacted
+- [ ] iOS
+- [ ] macOS
+
+### Description of changes
+
+(a summary of the changes made, often organized by file)
+
+### Verification
+
+(how the change was tested, including both manual and automated tests)
+
+| Before                                       | After                                      |
+|----------------------------------------------|--------------------------------------------|
+| Screenshot or description before this change | Screenshot or description with this change |
+
+### Pull request checklist
+
+This PR has considered:
+- [ ] Light and Dark appearances
```

Comment:
iPhone/iPad check for iOS: there were few regression happen when people only tested iPhone .... (regular compact?)

Reply:
I'm trying to come up with a better way to phrase it that works more generically for macOS as well. @anandrajeswaran any ideas? 

Worst case we can just add "Size classes and scaling (e.g. compact on iPhone vs regular on iPad, various window sizes on macOS)".

Reply:
What do you think about the new wording?

- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)